DayrSuryan/DSEthiop3.xml

```diff
@@ -759,9 +758,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <layoutDesc>
                            <layout columns="1" writtenLines="21 24">
                                 <locus from="101v" to="102v"/>
-                                <note corresp="#textarea2">Data on text area dimensions taken from <locus target="#101v"/>.</note>
```

Comment:
the corresp and the xml:id must correspond (which indeed they didn't...), I would propose textarea2. There are no margin measurements, so #margins can be deleted.

Reply:
ok